cartographer/mapping/proto/2d/grid_2d.proto

```diff
@@ -34,6 +35,7 @@ message Grid2D {
   CellBox known_cells_box = 3;
   oneof grid {
     ProbabilityGrid probability_grid_2d = 4;
+    TSDF2D tsdf_2d = 5;
```

Comment:
hm - why was 5 unused? But I also don't see a comment saying that it is reserved due to previous usage. So maybe it's ok?

Reply:
It was intentionally unused for the upcoming TSDF integration.